### DISCUSSION TITLE
DevOps Engineer @ InoVirtue ~27.5k - 30k

### DISCUSSION THREAD

**Discussion by krukowskid** [AzureCommunityPL/group#437] [Oferty pracy]
opis formalny: https://justjoin.it/offers/inovirtue-devops-engineer

opis nieformalny:
Stack: 
dotnet 6
react
Azure (App Service - w trakcie planowania przejścia na AKS, static web apps, cosmosdb - gremlin, service bus - jeszcze w powijakach + TF, ale jak ktoś woli Bicepa to dla mnie ekstra - można rozważyć zmianę)
launchdarkly (feature flags)
GitHub + GitHub Actions - pełne CICD i każdy commit do maina idzie na proda

Forma współpracy - na ten moment tylko B2B bezpośrednio do firmy (Niemcy), ale docelowo jak projekt się rozwinie ma być większa swoboda pod tym względem.
Lokalizacja: Full remote, ale jak mieszkacie w Monachium to można wpadać do biura. Raz w miesiącu bierzemy biuro na jeden dzień w Krakowie, żeby pogadać na żywo, ale nie jest to obowiązkowe.
Zespół techniczny w większości PL, ale generalnie ludzie z całej EU, więc angielski komunikatywny wymagany,

Dopiero co wypuściliśmy MVP,
realny wpływ na to co się dzieje 
"Urlop" jest, wynagrodzenie takie jak w tytule w zryczałtowanej formie/mc (nie z timesheetu), nikt nie rozlicza z patrzenia w monitor, a z wykonanej roboty - jak idzie do przodu to wszyscy są zadowoleni, ale siedzenie po godzinach nie jest mile widziane :)

Zainteresowanych, ale niezdecydowanych zapraszam do pytania w dyskusji poniżej, albo na linkedin: https://www.linkedin.com/in/piotr-krukowski/

**Comment by smereczynski** [comment]
Cześć,
Uzupełnij proszę wymagane elementy ogłoszenia.

**Reply by krukowskid** [reply]
poprawione, dzięki

**Comment by wi5nia** [comment]
Prosimy o niezwlozczne poprawienie posta zgodnie z wytycznymi pojawiajacymi sie podczas zakladania nowego watku (reguly sa w opisie kategorii) albo post zostanie usuniety.

**Reply by krukowskid** [reply]
panowie, poratujcie odnośnikiem. Przed dodaniem szukałem średnio intensywnie, teraz szukam mocno intensywnie i nie mogę dotrzeć do reguł dotyczących ogłoszeń

**Reply by wi5nia** [reply]
W opisie kategorii masz proste wytyczne 
![image](https://user-images.githubusercontent.com/12593852/171403972-5be928fd-7a75-41ea-9108-a59025974b23.png)

**Reply by smereczynski** [reply]
Jak dodajesz wpis to przy wyborze kategorii masz opis wymaganych informacji.

**Reply by krukowskid** [reply]
Ok, wszedłem najpierw w kategorię, a potem nacisnąłem "New Discussion" - miałem już uzupełnioną kategorię i dlatego przeoczyłem ten opis. Dzięki - poprawiam

**Reply by wi5nia** [reply]
Czekamy az pojawia sie szablony dla dyskusji tak jak sa dla issues w GH, ale poki co nie znamy ETA :( wtedy bedzie duzo jasniej.

**Reply by krukowskid** [reply]
Poprawione! Kilka razy przejrzałem faq, code of conduct i zacząłem nawet w wyszukiwarce po słowach klucz, więc wybaczcie :D

**Reply by wi5nia** [reply]
A gdzie widelki/kwota?

**Reply by krukowskid** [reply]
jest w tytule ogłoszenia ~27.5k - 30k
